include/mbedtls/x509_crt.h

```diff
@@ -651,4 +651,9 @@ int mbedtls_x509write_crt_pem( mbedtls_x509write_cert *ctx, unsigned char *buf,
 }
 #endif
 
+#ifndef MBEDTLS_TLS_DEFAULT_ALLOW_SHA1
+/* The test infrastructure requires a positive define */
+#define MBEDTLS_X509__DEFAULT_FORBID_SHA1
+#endif
```

Comment:
We probably want to extend our test infrastructure to handle that at some point, but most probably *not* a few days before a release. I suggest keeping Gilles' solution for now add creating a separate ticket for extending the test infra.